### PR TITLE
Vilnius 2019 blog post announcing schedule and events and activities

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/vilnius/2019/news/announcing-schedule-activities.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Due to catering and printing requirements, we will freeze ticket sales on **Monday, 27th May**. If you register after this date, we will not be able to print a badge for you (although you probably don't need to worry about the food!).
+
+Events and activities
+---------------------
+
+Write the Docs is more than a conference. At each of our events, we try to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge.
+
+Here is the overview of what you can expect during the conference:
+
+* **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *9:30am*. Please email us if you want to submit a project for the Writing Day!
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/news/announcing-schedule-activities.rst]:
```suggestion
* **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *10:30*. Please email us if you want to submit a project for the Writing Day!
```

**Comment by thatdocslady** [comment]
@aidiss do you have the details of the welcome reception venue yet? also, what is the deal with drinks, can we pay for this?

File: docs/conf/vilnius/2019/news/announcing-schedule-activities.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+We'll also be running the `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ on Sunday and `lightning talks <https://www.writethedocs.org/conf/vilnius/2019/lightning-talks/>`_ after lunch on Monday.
+
+You'll be able to sign up for Writing Day projects and lightning talks during the conference.
+
+Tickets are going!
+------------------
+
+We still have some tickets available, so head over to the `Tickets <https://www.writethedocs.org/conf/vilnius/2019/tickets/>`_ page and grab one!
+
+Due to catering and printing requirements, we will freeze ticket sales on **Monday, 27th May**. If you register after this date, we will not be able to print a badge for you (although you probably don't need to worry about the food!).
```
Comment (mxsasha) [inline docs/conf/vilnius/2019/news/announcing-schedule-activities.rst]: I take it this means you can't buy a ticket anymore, but you can still register it to your name, after this date? Or not? I am confused by the combination of "freeze sales" with "register after this date".

File: docs/conf/vilnius/2019/news/announcing-schedule-activities.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Events and activities
+---------------------
+
+Write the Docs is more than a conference. At each of our events, we try to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge.
+
+Here is the overview of what you can expect during the conference:
+
+* **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *10:30*. Please email us if you want to submit a project for the Writing Day!
```
Comment (mxsasha) [inline docs/conf/vilnius/2019/news/announcing-schedule-activities.rst]: "please email us" sounds contradictory with the earlier "you can sign up at the conference" ?

**Review by mxsasha — CHANGES_REQUESTED** [review]
Great work, left two minor notes :)

File: docs/conf/vilnius/2019/news/announcing-schedule-activities.rst (hunk 2 above)
Comment (thatdocslady) [inline docs/conf/vilnius/2019/news/announcing-schedule-activities.rst]:
```suggestion
Keep in mind that due to printing schedules, if you register after **Monday, 27th May** we will not be able to print a badge for you.
```

File: docs/conf/vilnius/2019/news/announcing-schedule-activities.rst (hunk 3 above)
Comment (thatdocslady) [inline docs/conf/vilnius/2019/news/announcing-schedule-activities.rst]:
```suggestion
* **Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ will be held on Sunday from *10:00-18:00*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *10:30*. You can sign up for a project during the day, or email us in advance if you want to have your project listed on our website.
```

**Review by thatdocslady — COMMENTED** [review]
Thanks @mxsasha, I made a few suggestions based on your feedback. LMK :)

**Review by mxsasha — APPROVED** [review]

**Comment by aidiss** [comment]
> 
> 
> @aidiss do you have the details of the welcome reception venue yet? also, what is the deal with drinks, can we pay for this?

Not yet.
Yes, we can pay for drinks. Why that should be a problem?

**Review by aidiss — APPROVED** [review]
It's good, we can update later if needed.

**Comment by thatdocslady** [comment]
@aidiss there is no problem with paying for drinks if the venue budget requirements are reasonable. This is why I ask, because it depends on how your budget spreadsheet looks like. Please update the numbers and check with the venue, I won't publish details until we have a deal closed with the venue.

And please keep in mind, that even though the blog post can be updated, we send out emails from Mailchimp to our mailing list, and *those* can't be edited later. This is why we need to do a review on each blog post before we send it.

**Pull request merged**